### DISCUSSION TITLE
Node.js: Auth on server

### DISCUSSION THREAD

**Discussion by vvmspace** [gram-js/gramjs#727] [Ideas]
**To auth user on server I need `2` endpoints:**
1. Auth init: request code
2. Auth complete: user sends code

**But we have all auth in one method:**
```
client.start({
phoneNumber: ...,
password: ...,
phoneCode: ...
})
```

How to do it separately?

**Comment by ghost** [comment]
Did you figure this out?

**Comment by guang8886667** [comment]
帮助申请telegram application 秒成功 无论什么账号 解决telegram application error问题

Help apply for telegram application in seconds, no matter what account, solve telegram application error problem

请联系Telegram ： @guang8886667

请联系Telegram : https://t.me/guang8886667
